TTHAnalysis/macros/prepareTHQEventVariableFriends.py

```diff
@@ -0,0 +1,403 @@
+#!/usr/bin/env python
```

Comment:
There shouldn't be a need of a different prepareEventVariables.
Can you implement what you need in the standard one?